instructure/canvas-ios
# Fix grading period selector

refs: MBL-16039
affects: Student, Teacher
release note: Fixed selecting a grading period in assignments list not updating the screen.

test plan:
- Go to a course with assignments distributed amongst multiple grading periods.
- On the assignments list switch between grading periods.
- Updated assignments list should match what can be seen on the website.
--
- Enter assignments list without any grading period filter setup.
- Go to web, un-publish an assignment.
- Pull to refresh assignments list.
- Assignment should disappear.
--
- Go to course, enter assignments list.
- Loading state should be visible instead of empty panda upon screen appearance.
